tapdb/multiverse.go

```diff
@@ -31,7 +32,8 @@ type (
 type BaseMultiverseStore interface {
 	BaseUniverseStore
 
-	UniverseRoots(ctx context.Context) ([]BaseUniverseRoot, error)
+	UniverseRoots(ctx context.Context,
```

Comment:
I think `sqlc.UniverseRootsParams` should get a type alias, like `BaseUniverseRoot` above.